packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -38,8 +38,15 @@ const LowBalanceMessage = () => (
 );
 
 const BuySellModalFooter = ({ onCancel, is_submit_disabled, onSubmit }) => {
+    const { my_profile_store } = useStores();
     return (
-        <React.Fragment>
+        <div
+            className={
+                my_profile_store.should_show_add_payment_method_form
+                    ? 'add-payment-method__footer'
+                    : 'buy-sell__modal-footer'
+            }
+        >
```

Comment:
`nit` not really crucial, but just adding it here in case others will comment on it:
```
className={classNames({
   'add-payment-method__footer': my_profile_store.should_show_add_payment_method_form,
   'buy-sell__modal--footer': !my_profile_store.should_show_add_payment_method_form
})}
```

Reply:
IMO thats better, i'll just do some quick changes for both then